README.md

```diff
@@ -331,8 +331,31 @@ The IP configuration supports the following options:
 
     Manual DNS configuration can be specified via a list of addresses
     given in the `dns` option, a list of domains to search given in the
-    `dns_search` option and a list of dns options to set given in the
-    `dns_options`.
+    `dns_search` option and a list of DNS options to set given in the
+    `dns_options`. Currently, `dns_search` and `dns_options` are only
+    supported for IPv4 nameservers. And DNS options are only supported
```

Comment:
```suggestion
    supported for IPv4 nameservers. DNS options are only supported
```

Reply:
I changed into  "`dns_options` is only supported for the NetworkManager provider and IPv4
    nameservers."